PlayerCharacter.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using UnityEngine;
+
+class PlayerCharacter{
+  public string name;
+  public string className;
+  private string race;
+  private string subrace;
+  private string background;
+  
+  private int strength;
+  private int wisdom;
+  private int dexterity;
+  private int constitution;
+  private int intelligence;
+  private int charisma;
+  private int level;
+  private int maxhp;
+  public int currenthp;
+  public int armorClass;
+  private int initiative;
+  private int speed;
+  public currentSpeed;
+
+  private int strSaveProficiency;
```

Comment:
Proficiencies should be Booleans.

Reply:
I made them into bool, kept proficiencyBonus as int.